cuegui/cuegui/SubscriptionGraphWidget.py

```diff
@@ -0,0 +1,216 @@
+import sys  # We need sys so that we can pass argv to QApplication
```

Comment:
Could you explain more about how this works? Kind of weird to see it imported but not actually used anywhere in the file.

Reply:
doesn't sound right. I'll remove it as I can't see a reason for it.